lib/main_table.json

```diff
@@ -0,0 +1,12 @@
+{
+    "AttributeDefinitions": [
+        {"AttributeName": "index", "AttributeType": "S"}
```

Comment:
yea, my problem with id is that this string is made of two other things called id so it gets confusing to keep calling things id.

What about `key`?